library/core/src/num/dec2flt/mod.rs

```diff
@@ -204,155 +194,69 @@ impl ParseFloatError {
     }
 }
 
-#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Why was this removed? You can't remove stable items.

Reply:
Accident, my bad. Thanks for catching this.

Reply:
Fixed as of [fb1a817fa13f774ce4da27fb9769db2779163030](https://github.com/Alexhuszagh/rust/blob/fb1a817fa13f774ce4da27fb9769db2779163030/library/core/src/num/dec2flt/mod.rs#L197-L202).